lib/index.ts

```diff
@@ -30,12 +30,19 @@ export function loadConfig(
 
   options = options || {};
   const secretConfig =
-    options.secretConfig || path.resolve(dir, 'config.secret.json');
+    options.secretConfig ||
+    process.env['CONFIG_SECRET_FILE'] ||
+    path.resolve(dir, 'config.secret.json');
 
   if (!path.isAbsolute(dir)) {
     throw new Error('config requires absolute path to read from');
   }
 
+  const serviceEnv = process.env['SERVICE_ENV'];
+  const localConfig = serviceEnv ? serviceEnv : 'local';
```

Comment:
it TS 3.7 this can be shortened to 
```
const localConfig = serviceEnv ?? 'local';
```

:parrot-parrotdad:

Reply:
`??` appears to behave differently for e.g. empty string, so it's not just a shortening.

It might be more correct here, but I'm .. going to leave it as is, I think.

```
const a = '';
console.log(a ?? 'local');
```
->
```
const a = '';
console.log((a !== null && a !== void 0 ? a : 'local'));
```

... which would be `'local'` before and `''` after, for `''` input?